### ISSUE TITLE
gcode generation does not respect svg clipping

### ISSUE THREAD

**Issue by bvarner** [sameer/svg2gcode#66]
Having an SVG where a shape in front of another is used as a clipping region for a shape behind it, the gcode will be generated for the entire shape behind the clip region, rather than respecting the clip region.

This is really strange, as the rendering on the webpage respects the clip. :-(

**Comment by sameer** [comment]
Yeah the preview can be misleading, thanks for calling this out. FWIW you should be able to flatten the clip in inkscape.
